RequestLogger/RequestLogger.cs

```diff
@@ -53,6 +70,21 @@ public RequestLogger(RequestDelegate next, IOptions<RequestLoggerOptions> option
 
         public static int RequestsPerSecond => RPS.Count > 0 ? RPS[0] : counter;
 
+        /// <summary>
+        /// Return the path and query string if it exists
+        /// </summary>
+        /// <param name="request">HttpRequest</param>
+        /// <returns>string</returns>
+        public static string GetPathAndQuerystring(HttpRequest request)
```

Comment:
I'm wondering if this should be in a different class; maybe a static utils class for this and other static methods that come up but have no home?  It feels like an odd dependency to need the logger to help parse the path and query string.

Reply:
agree - added a TODO and will move in next PR